packages/components/src/FormField/FormField.css

```diff
@@ -232,6 +238,13 @@
   -webkit-box-shadow: 0 0 0 30px var(--color-surface) inset !important;
 }
 
+/* If there's no label, add back padding-top to input */
```

Comment:
personally I think the code already does a solid job of communicating this. `:not(:has(.label` and then the corresponding rule is adding the padding

is there something you think the comment adds or better explains?